cloud/releases/cloud-release-notes.md

```diff
@@ -0,0 +1,44 @@
+---
+id: cloud-release-notes
+title: Centreon Cloud release notes
+---
+
+## XX October/November, 2023
+
+### Centreon IT Edition
+
+* **New notification configuration system**:
+
+   You can now configure notifications on a per user and per resource basis.
+Associate users with resources and the statuses for which users will be notified, define a notification format, and tailor notifications according to your users' scope of monitoring. Only email notifications are available now (more to come).
+
+* **Resources Status**:
+
+   - In addition to the current view (**All** view), we have added 2 new views allowing users to see only the services
+  (**By Services** view), as well as a view to see the hosts and all their attached services (**By Hosts** view).
+  These two new views have been designed based on what was done in deprecated pages.
+   - We have also completely redesigned filters, making it easier to select resources.
+
+* **Dashboards Beta**:
+
+   With Dashboards, effortlessly create and share intuitive dashboards for real-time monitoring of metrics, host/service statuses, and much more. Join our [Dashboards Beta group](https://thewatch.centreon.com/groups/centreon-dashboards-beta-59) on our community platform The Watch!
```

Comment:
Hello @rbignotticentreon, pourquoi mentionner le groupe The Watch ici dans la RN cloud et pas dans la RN on-prem également ?